modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointMetricsTrackerTest.java

```diff
@@ -17,16 +17,14 @@
 
 package org.apache.ignite.internal.pagememory.persistence.checkpoint;
```

Comment:
Noticed that we have tests for checkpoint metrics tracker, but not for compaction. Not sure if they're really needed though